skyfield/sgp4lib.py

```diff
@@ -153,6 +153,30 @@ def ITRF_position_velocity_error(self, t):
         rITRF, vITRF = TEME_to_ITRF(t.ut1, rTEME, vTEME)
         return rITRF, vITRF, error
 
+    def is_sunlit(self, ephemeris, times):
+        """Calculates if the satellite is currently sunlit (True/False) at each time in times. Returns an array of booleans."""
+        Re = 6378.137  # Equatorial radius of the Earth in km
+        earth = ephemeris['earth']
+        sun = ephemeris['sun']
+        sat = earth + self
+
+        sunpos, earthpos, satpos = [
```

Comment:
Let's pivot these from a list to a plain set of 3 assignments. That way the correspondences are clearer, the code will run faster, and it will take as many lines of code anyway.